profile/README.md

```diff
@@ -1,15 +1,13 @@
 <!-- markdownlint-disable MD041 -->
-## Hexa Programming Language
+## Welcome to the **Hexa** organization
```

Comment:
Do not use "organization"
Change to something more fun and friendly, a warming welcome